deploy/kubedirector/crd-settings.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: kubedirectorsettingss.kubedirector.bluedata.io
+spec:
+  group: kubedirector.bluedata.io
+  names:
+    kind: KubeDirectorSettings
+    listKind: KubeDirectorSettingsList
+    plural: kubedirectorsettingss
+    singular: kubedirectorsettings
+  scope: Namespaced
+  version: v1alpha1
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          properties:
+            storageClassName:
```

Comment:
Should we call this `defaultStorageClassName`?

Reply:
ok